### PR TITLE
Design Upgrade

### PR DESCRIPTION
**Pull request by itubetw**
just looking better xD

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,49 +7,57 @@
     <meta content="Custom ddnet based teeworlds client chillerbot-ux with warlist" name="description">
     <title>chillerbot-ux ddnet client</title>
     <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap" rel="stylesheet">
```
Comment (ChillerDragon) [inline index.html]: Not a big fan of external urls. Especially google. If you need a font it should be served by the site.

File: style.css [hunk 2]
```diff
@@ -1,51 +1,279 @@
...
+}
+
+.twinkling {
+    background: transparent url(http://www.script-tutorials.com/demos/360/images/twinkling.png) repeat top center;
+    z-index: -2;
+    animation: move-twink-back 200s linear infinite;
+}
+
+.clouds {
+    background: transparent url(http://www.script-tutorials.com/demos/360/images/clouds3.png) repeat top center;
```
Comment (ChillerDragon) [inline style.css]: Same here. If you need a image put it in the img folder. Also that does not look like a teeworlds image.

File: index.html [hunk 3]
```diff
@@ -7,49 +7,57 @@
...
+                    </div>
+                </div>
+            </div>
+        </div>
         <div class="footer">
             <div class="small_tree"></div>
             <div class="bush"></div>
             <div class="grass"></div>
         </div>
-    </div> <!-- all -->
```
Comment (ChillerDragon) [inline index.html:53]: why was this removed?

**Review by ChillerDragon — CHANGES_REQUESTED** [review]
I like the old design not sure if I want some changes. Could you attach some screenshot?

**Comment by itubetw** [comment]
> I like the old design not sure if I want some changes. Could you attach some screenshot?

image

just looking more modern (zoomed out little bit)

File: style.css [hunk 4]
```diff
@@ -1,51 +1,240 @@
...
+        clip-path: inset(46% 0 34% 0);
+        transform: translate(5px, -5px);
+    }
+    100% {
+        clip-path: inset(100% 0 0% 0);
+        transform: translate(0);
+    }
+}
+
+@keyframes rotate-sun {
```
Comment (ChillerDragon) [inline style.css]: which sun

**Comment by ChillerDragon** [comment]
Oh yea that new design looks smoother. Something about the buttons bothers me (but I am no design so idk what) should be fine for now tho.

I liked the big sun it made it so teeish. Now it looks a bit plain. Do you think it would fit in with the new design?

Could you download the google font and put it into a new fonts/ directory and import that from css instead of the google endpoint. Then I would merge it.

**Comment by itubetw** [comment]
I added the sun in the background and I think it would be better the way it looks now. it's just a lot cleaner. But hopefully in the future the next chillerbot-ux release would make it even more beautiful, perhaps with a moving sun. I did what you requested.

**Comment by ChillerDragon** [comment]
Thanks
